test/stages/__init__.py

```diff
@@ -22,7 +22,7 @@
 )
 
 
-def add_sg(ds, id, external_id: str):
+def add_sg(ds, id, external_id: str) -> SequencingGroup:
```

Comment:
All changes in the test setup is just reflecting that we no longer create cohorts, create datasets in those cohorts, then create SGs.